docs/agents/tailwind-plus-rabbita.md

```
@@ -0,0 +1,617 @@
+# Agent Guide: Porting Tailwind Plus to Rabbita
+
+This project uses **MoonBit + Rabbita** (The Elm Architecture), not React.
+
+---
+
+## Reference material
+
+Licensed Tailwind Plus snippets live under:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)